### PR TITLE
Update Ember best practices following 1.13 standards

### PR DESCRIPTION
**Pull request by matteodepalo**

### CODE REVIEW

File: best-practices/ember/README.md [hunk 1]
```diff
@@ -34,19 +34,18 @@ Based on the routes above, the corresponding view structure is this one:
...
 
 ## Coding style and best practices
 
 ### Templates
-A `render` statement should always refer to a view, never directly to a template.
-Templates should not contain the root element with the main view class. Add the main class to the `classNameBindings` property in the view file.
-For components, the main class should be the same as the component name, minus the `as-` prefix. For views, it should be the same as the view name.
+Templates not backed by a component should always contain the root element with the main class named after the template.
+
+For components, add the main class to the `classNameBindings` property. The class should be the same as the component name, minus the `as-` prefix.
```
Comment (eugeniodepalo) [inline best-practices/ember/README.md]: We remove the `as-` prefix in all our components, but I think that this shouldn't be the followed rule anymore. I think we should have `.as-` css classes

Comment (matteodepalo) [inline best-practices/ember/README.md]: :+1:

File: best-practices/ember/README.md [hunk 2]
```diff
@@ -34,19 +34,18 @@ Based on the routes above, the corresponding view structure is this one:
...
+Styles should follow exactly the same folder structure as their corresponding components. This is not required by any naming convention, but helps greatly to quickly find the styles for a certain section of the application. It follows that each component should have an unique class name (plus optional tag name) assigned to it.
 HTML elements global styling (a.k.a reset) should be put inside the `styles/_default.scss` file.
 
 ## Coding style and best practices
 
 ### Templates
-A `render` statement should always refer to a view, never directly to a template.
-Templates should not contain the root element with the main view class. Add the main class to the `classNameBindings` property in the view file.
-For components, the main class should be the same as the component name, minus the `as-` prefix. For views, it should be the same as the view name.
+Templates not backed by a component should always contain the root element with the main class named after the template.
```
Comment (eugeniodepalo) [inline best-practices/ember/README.md]: This is true only for "routable" templates, so the root templates like application/projects/etc. So maybe we could explain that.

File: best-practices/ember/README.md [hunk 3]
```diff
@@ -72,40 +71,16 @@ Whenever you need a wrapper for a certain element, unless absolutely necessary,
 
 ---
 
-Use the least amount of wrapper elements possible.
-
-**Do this:**
-
-```
-{{#as-scrollable tagName="nav"}}
```
Comment (eugeniodepalo) [inline best-practices/ember/README.md]: I feel like this is still relevant.

File: best-practices/ember/README.md (hunk 2 above)
Comment (matteodepalo) [inline best-practices/ember/README.md]: I say "not backed by a component" to express that. Do you think we should say "routable" templates?

File: best-practices/ember/README.md (hunk 3 above)
Comment (matteodepalo) [inline best-practices/ember/README.md]: The problem is that the as-scrollable usage has changed so now we wrap the component with <nav>. Shall we use another component?

File: best-practices/ember/README.md (hunk 2 above)
Comment (eugeniodepalo) [inline best-practices/ember/README.md]: Maybe! We should somehow make it clear that if you want to have a template without a component then you can just do a partial instead.

**Comment by eugeniodepalo** [comment]
:+1:

**Comment by daemonsy** [comment]
:heart_eyes: :trophy:

**Pull request merged**